### ISSUE TITLE
I cannot see the offline list on the Bluzelle Discord

### ISSUE THREAD

**Issue by njmurarka** [mushroomgenie/DiscordBluzelleBot#4]

**Comment by mushroomgenie** [comment]
What do you mean by this?

**Comment by SatyaSnehith** [comment]
Did you connect the bot to discord with of invite link?
![image](https://user-images.githubusercontent.com/33853823/126483627-270fddc5-b64f-41ee-92b9-c7466b29908a.png)

**Comment by njmurarka** [comment]
Yes... but is there some reason Discord does not show the offline users? Try to goto Bluzelle's Discord and you might see what I mean.

**Comment by SatyaSnehith** [comment]
This might be one of the reasons that with more users it will be harder for the server to display all the members.
